hooks/post/10-post_install.rb

```diff
@@ -18,7 +18,7 @@ def proxy?
       Kafo::Helpers.certs_generate_command_message
     elsif Kafo::Helpers.module_enabled?(@kafo, 'katello_devel')
       Kafo::Helpers.dev_server_success_message(@kafo)
-      Kafo::Helpers.new_install_message(@kafo) if @kafo.param('foreman', 'authentication').value == true && new_install?
+      Kafo::Helpers.new_install_message(@kafo) if @kafo.param('foreman', 'authentication').try(:value) == true && new_install?
```

Comment:
Isn't try a rails thing rather than a base ruby one? I also think I wasn't thinking while I wrote this line. the devel scenario doesn't use the foreman module so it'll never be true.

Reply:
So, we can safely remove this line completely?

Reply:
I think https://github.com/Katello/katello-installer/pull/600 is a better solution.